deployer/src/config_creator.py

```diff
@@ -210,6 +210,28 @@ def to_larecipe_config(config, urls=None):
     return config
 
 
+def to_publii_config(config, urls=None):
```

Comment:
I think there is a typo

Reply:
Regarding the double `i` ? It is actually the original name https://getpublii.com/dev/